docs/docs/reference/crd-reference/appcontext.md

```diff
@@ -0,0 +1,163 @@
+---
+comments: true
+---
+
+# KeptnAppContext
+
+The `KeptnAppContext` custom resource
+allows you to add metadata and links to traces for a specific application.
+This enables you to enrich your traces with additional information,
+making it easier to understand and analyze
+the performance of your applications.
+
+It works with the corresponding
+[KeptnApp](app.md) resource
+that has the same name and is located in the same namespace.
+
+## Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1beta1
+kind: KeptnAppContext
+metadata:
+  name: <app-name>
+  namespace: <app-namespace>
+spec:
+  metadata:
+    <custom-attributes>
+    spanLinks: "<link>
```

Comment:
the `spanLinks` field is **next to** the `metadata` field, not inside of it. And it's a list of links.
```suggestion
  spanLinks:
  - "<list of links>"
```